test/image-blueprints/group2/rhel92-source-fake-next-minor.toml

```diff
@@ -1,5 +1,5 @@
 name = "rhel-9.2-microshift-source-fake-next-minor"
-description = "A RHEL 9.2 image with the RPMs built from source from the current PR but with the _version_ set to the next y-stream."
+description = "A RHEL 9.2 image with the RPMs built from source from the current PR but with the _version_ set to the current+2 y-stream."
```

Comment:
I think you updated the description in the wrong file here?

Reply:
Fixed